ush/forecast_postdet.sh

```diff
@@ -204,8 +204,10 @@ EOF
   if [[ "${QUILTING}" = ".true." ]] && [[ "${OUTPUT_GRID}" = "gaussian_grid" ]]; then
     local FH2 FH3
     for fhr in ${FV3_OUTPUT_FH}; do
-      FH3=$(printf %03i "${fhr}")
-      FH2=$(printf %02i "${fhr}")
+      fhrf=$(echo "${fhr}" | bc -l)
+      fhri=$(printf %.0f "${fhrf}")
```

Comment:
Can you explain what is the issue this seems to be solving?

Reply:
If `fhr` is a decimal number, it needs to be converted to a whole number here because we want to use a whole number in the filename of the output data. The first `fhr` will be a decimal number in cases where `OFFSET_START_HOUR > 0`. For example, if `OFFSET_START_HOUR` is `3`, the first `fhr` will need to be `${OFFSET_START_HOUR}+(${DELTIM}/3600)`, or 3.0833 if `DELTIM` is 300. If we keep `fhr` as 3, then output data for `f003` will not be generated.  I can add a brief comment to the script here.